test/server.js

```diff
@@ -13,9 +13,61 @@ const config = {
     port: 5432,
     name: 'sensorthingstest',
     user: 'postgres',
-    pass: '12345678'
+    pass: '12345678',
+    hooks: {
+      'beforeValidate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeValidate: 'beforeValidate'
+        });
+      },
+      'afterValidate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterValidate: 'afterValidate'
+        });
+      },
+      'beforeCreate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeCreate: 'beforeCreate'
+        });
+      },
+      'afterCreate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterCreate: 'afterCreate'
+        });
+      },
+      'beforeDestroy': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeDestroy: 'beforeDestroy'
+        });
+      },
+      'afterDestroy': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterDestroy: 'afterDestroy'
+        });
+      },
+      'beforeRestore': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeRestore: 'beforeRestore'
+        });
+      },
+      'afterRestore': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterRestore: 'afterRestore'
+        });
+      },
+      'beforeUpdate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeUpdate: 'beforeUpdate'
+        });
+      },
+      'afterUpdate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterUpdate: 'afterUpdate'
+        });
+      }
+    }
   }
```

Comment:
Just don't use `server.js`. Create a dedicated express app for the hook tests inside `test_client_db_hooks.js`